pkg/apis/edge/v2alpha1/edge-placement.go

```diff
@@ -74,6 +74,7 @@ type EdgePlacement struct {
 
 // EdgePlacementSpec holds a desired binding between (a) a set of Locations and
 // (b) a way of identifying objects to downsync and a way of identifying objects to upsync.
+// An object that matches both the downsync and the upsync is upsynced (i.e., upsync has precedence).
```

Comment:
So this object is upsynced **only**, not downsynced? May I know why upsync takes precedence? (not against the idea at all, just want to understand the thoughts/reasoning behind it)

Reply:
Remember that we define the jargon very carefully. "downsynced" and "upsynced" are adjectives that apply to an object, _not_ the past tense of a verb. A downsynced object has state flowing in both directions: desired state from core to WEC, reported state from WEC to core.

In the quest to make it easy for a user to identify all the objects that she wants downsynced, we inevitably make the user cast a wide net. If there **are** any objects to be upsynced, it is not unlikely that they will be a subset of those that match the downsync criteria.

Reply:
I see. Thanks for your help to understand this.